var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -76,6 +76,8 @@ def install(self, spec, prefix):
 
         if '+libiberty' in spec:
             configure_args.append('--enable-install-libiberty')
+            configure_args.append('--disable-nls')
```

Comment:
Thanks @kserradell that would be great! I especially appreciate it since you'd have to go to the effort of creating another PR :-)